portal-api/src/v1/users/sign-in-link.service.js

```diff
@@ -113,6 +113,10 @@ class SignInLinkService {
   }
 
   async #sendSignInLinkEmail({ userEmail, userFirstName, userLastName, signInLink }) {
+    if (process.env.NODE_ENV === 'development') {
+      console.info('🔗 signInLink 🔗', signInLink);
```

Comment:
Is there a reason to not just `return` after `console.info(...)` so that we don't even bother emailing the sign in link?

Reply:
I didn't want to interfere with the normal flow at all, even if it fails